test/gql_spec.js

```diff
@@ -1,20 +1,606 @@
-/* globals describe, it */
-var gql = require('../lib/gql'),
-    knex = require('knex')({}),
-    toSQL;
+var _, chai, knex, gql;
+_ = require('lodash');
 
-toSQL = exports.toSQL = function (input, resource) {
-    var parsedFilter = gql.parse(input);
-    return gql.knexify(knex(resource), parsedFilter).toQuery();
-};
+chai = require('chai');
+chai.should();
+chai.use(require('chai-bookshelf'));
+chai.use(require('chai-as-promised'));
+
+knex = require('knex')({
+    client: 'sqlite3',
+    connection: {filename: ':memory:'}
+    // , debug: true
+});
+
+gql = require('../src/gql');
 
 describe('GQL', function () {
-    it('should correctly get from GQL -> SQL', function () {
-        toSQL('id:1', 'posts').should.eql('select * from "posts" where "posts"."id" = 1');
+    before(function (done) {
+        var image = 'asdfghjkl;';
+        knex.schema.createTable('posts', function (table) {
```

Comment:
I wanted to test actual execution of queries beyond just parsing filters. So I wired up an in-memory sqlite database to test with.